test.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] Your test.md file is still in your repository. It should be ignored by .gitignore. :blush: It is happening because you have created your .gitignore file after creating the test.md file. :thinking: 
- For now you need to delete your test.md file from your local repository. :blush:
- Then create it again to check if the gitignore is ignoring it.
Please do the changes and make sure your gitignore should avoid this test.md file from going into your Github Repository :blush:

Reply:
Thanks for this wonderful feedback 👍 . The test.md file has been deleted from the repository